apps/system/utils/netcmd.c

```diff
@@ -95,6 +96,85 @@ struct ifconfig_cmd_info_s {
 extern int netdb_main(int argc, char *argv[]);
 #endif
 
+uint8_t * getIp(void)
```

Comment:
Could you remove a space and rename like below?
```
uint8_t *get_ip(void)
```
They are TizenRT coding rules.

Reply:
@j11591 Could you review this function? Is this a right location to locate?

Reply:
What is purpose of this function???
Just get IP???? If it is right, there are lots of meaning less ioctl.
And please checking "netlib_get_ipv4addr" in external.

Reply:
Dear @j11591,
Yes, this function is just to get IP, i will remove this function and use "netlib_get_ipv4addr" instead.